Bio/LogisticRegression.py

```diff
@@ -17,8 +17,9 @@
 """
 
 
-import numpy
-import numpy.linalg
+import numpy as np
+
+# import np.linalg
```

Comment:
I think you can remove this comment.

(I'm assuming this still works - there probably was once a reason to explicitly import linalg)